lib/ui/window/platform_configuration.h

```diff
@@ -197,6 +198,44 @@ class PlatformConfigurationClient {
   ///
   virtual void RequestDartDeferredLibrary(intptr_t loading_unit_id) = 0;
 
+  //----------------------------------------------------------------------------
+  // API exposed as FFI calls in Dart.
+  //
+  // These are probably not supposed to be called directly, and should instead
+  // be called through their sibling API in `PlatformConfiguration` or
+  // `PlatformConfigurationClient`.
+  //
+  // These are intentionally undocumented. Refer instead to the sibling methods
+  // above.
+  //----------------------------------------------------------------------------
+
+  static std::string DefaultRouteNameOrThrow();
```

Comment:
These static methods were originally namespace scoped functions private to `platform_configuration.cc`

Do these functions now need to have externally visible declarations so they can be called from the resolver in `lib/ui/dart_ui.cc`?

If so, then I'd prefer to create a dedicated class for that purpose that is separate from the `PlatformConfigurationClient` interface class.

Reply:
I've moved all these into their own class: `PlatformConfigurationNativeApi`.